src/components/MultiSelectQuizQuestion.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+
+const MultiSelectQuizQuestion = ({
+  question,
+  choices,
+}: {
+  question: string;
+  choices: { text: string; isCorrect: boolean; explanation?: string }[];
+}) => {
+  const [selected, setSelected] = useState<boolean[]>(choices.map(() => false));
+  const [submitted, setSubmitted] = useState(false);
+  const [isCorrect, setIsCorrect] = useState<boolean | null>(null);
+
+  const handleChange = (idx: number) => {
+    setSelected((prev) => prev.map((val, i) => (i === idx ? !val : val)));
+  };
+
+  const handleSubmit = () => {
+    const correct = selected.every((val, i) => val === choices[i].isCorrect);
+    setIsCorrect(correct);
+    if (correct) setSubmitted(true);
+  };
+
+  return (
+    <div>
+      <p>{question}</p>
+      <ul style={{ listStyle: "none", padding: 0 }}>
```

Comment:
To match the other multiple choice, we need a gap (maybe gap-1?) on the outside container so that there's a gap between items